addon/serializers/json-api.js

```diff
@@ -93,6 +93,39 @@ var dasherize = Ember.String.dasherize;
 
   to the format that the Ember Data store expects.
 
+  **Customizing meta**
+
+  Since a JSON API Document can have meta defined in multiple locations you can
+  use the specific serializer hooks if you need to customize the meta.
+
+  One scenario would be to camelCase the meta keys of your payload. The example
+  below shows how this could be done using `normalizeArrayResponse` and
+  `extractRelationship`.
+
+  ```app/serializers/application.js
+  export default JSONAPISerializer.extend({
+
+    normalizeArrayResponse(store, primaryModelClass, payload, id, requestType) {
+      let normalizedDocument = return this._super(...arguments);
```

Comment:
Good example. Just a nitpick: shouldn't have a return here, should be:
`let normalizedDocument = this._super(...arguments);`

Same thing for `extractRelationship`